nix/pkgs/mantis.nix

```diff
@@ -0,0 +1,89 @@
+{ src
```

Comment:
I think that this is a duplicated file and should be dropped. @manveru wdyt?

Reply:
NB: the README and the `update-nix.sh` still create and refer to this.

Reply:
@manveru I pulled the latest develop and tried to run the updated script, but get this error:
```console
aakoshh@af-dp:~/projects/iohk/mantis$ ./update-nix.sh 
Determining new sha for sbt build, this can take several minutes to do a 'sbt compile'
error: don't know what to do with argument '.#mantis.deps.outputHash'
Try 'nix --help' for more information.
aakoshh@af-dp:~/projects/iohk/mantis$ nix --version
nix (Nix) 2.3.10
```
Do you know what I'm doing wrong?

Reply:
The build [failed](https://buildkite.com/input-output-hk/mantis/builds/2289#a8cc967c-bff2-4adc-9a5e-e678435158d1) as well in an unexpected way:
```

Determining new sha for sbt build, this can take several minutes to do a 'sbt compile'
--
  | error: value is null while a list was expected
  |  
  | at /nix/store/aaw01dbdp547458996y474ggivd4b7q6-source/nix/mantis.nix:24:6:
  |  
  | 23\|     captures = builtins.match ''.*version := "([^"]+)".*'' buildSbt;
  | 24\|   in builtins.elemAt captures 0;
  | \|      ^
  | 25\|
  | (use '--show-trace' to show detailed location information)
  | calculating hash failed!
```

Reply:
Added comments to the README about the need to install the Nix Flakes feature and updated the script to point at `version.sbt`.